server/api/person/person.js

```diff
@@ -24,6 +24,7 @@ const personSchema = new Schema({
   facebook: { type: 'String' },
   twitter: { type: 'String' },
   education: { type: 'String' },
+  job: { type: 'String', default: '' },
```

Comment:
as Job is optional it does not require a default: ''   use the same style as for education.

Reply:
Oh bless ive taken it out now !!! Thanks heaps